subsys/pm/device_runtime.c

```diff
@@ -61,7 +61,7 @@ static int runtime_suspend(const struct device *dev, bool async)
 		(void)k_work_schedule(&pm->work, K_NO_WAIT);
 	} else {
 		/* suspend now */
-		ret = pm->action_cb(pm->dev, PM_DEVICE_ACTION_SUSPEND);
+		ret = pm_device_action_run(pm->dev, PM_DEVICE_ACTION_SUSPEND);
```

Comment:
this patch should not be required now, device_runtime can easily stay away from `pm_device_action_run`.